### PR TITLE
Update readme and add method aliases to match ruby_gettext

### PR DESCRIPTION
**Pull request by rgraff**
Achieves near parity with ruby-gettext methods

### CODE REVIEW

File: lib/fast_gettext/translation.rb [hunk 1]
```diff
@@ -72,6 +76,14 @@ def ns_(*args, &block)
       # block is called once again to compare result
       block && translation == block.call ? translation : translation.split(NAMESPACE_SEPARATOR).last
     end
+    alias :nsgettext :ns_
+
+    def np_(*args, &block)
+      context, key = args[0..1]
+      nargs = ["#{context}#{CONTEXT_SEPARATOR}#{key}"] + args[2..-1]
```
Comment (grosser) [inline lib/fast_gettext/translation.rb]: CONTEXT_SEPARATOR does not exist

**Comment by grosser** [comment]
so many typos :D

File: lib/fast_gettext/translation.rb (hunk 1 above)
Comment (grosser) [reply lib/fast_gettext/translation.rb]: hmm also this breaks caching since now s and p use different caches ?

**Comment by grosser** [comment]
can you rebase this ?

File: spec/fast_gettext/translation_spec.rb [hunk 2]
```diff
@@ -141,7 +141,16 @@
...
+      ns_('Fruit|Apple','Apples',2).should == 'Apples'
+      ns_('Fruit|Apple','Apples',3).should == 'Apples'
+    end
+
+    it "translates pural with double namespace" do
+      # This behavior does not match gettext but
+      # let's make sure it continues to work as to not introduce
+      # a breaking change.
       ns_('Fruit|Apple','Fruit|Apples',2).should == 'Apples'
     end
```
Comment (grosser) [inline spec/fast_gettext/translation_spec.rb]: need np_ tests ?

File: lib/fast_gettext/translation.rb (hunk 1 above)
Comment (rgraff) [reply lib/fast_gettext/translation.rb]: commit to define CONTEXT_SEPARATER was just merged.

s and p do use different caches and can have different values. I'm not sure if that's by design but that's how ruby-gettext works.

```
msgctxt "Printer"
msgid "Open"
msgstr "translation-one"

msgid "Printer|Open"
msgstr "translation-two"
```

File: spec/fast_gettext/translation_spec.rb (hunk 2 above)
Comment (rgraff) [reply spec/fast_gettext/translation_spec.rb]: `np_` and the test are in another branch.

**Comment by rgraff** [comment]
Rebased once. But I need to rebase again after `np_` is merged in.

And then I need to implement `dp_` and `dnp_` for completeness.

**Comment by grosser** [comment]
the p_ change will need a major bump ... is there anything else you want to get in ?

**Comment by rgraff** [comment]
Hold on major bump until I can get `dp_` and `dp_`.  Might be later today or over weekend.

**Pull request merged**